certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json

```diff
@@ -795,14 +840,14 @@
                                                 "properties": {
                                                     "value": {
                                                         "type": "string",
-                                                        "description": "Admin Contact Street",
+                                                        "description": "Street address of the administrative contact",
                                                         "title": "Admin Contact Street"
                                                     },
                                                     "count": {
                                                         "type": "integer",
                                                         "format": "int32",
-                                                        "description": "Count",
-                                                        "title": "Count"
+                                                        "description": "Count of connected domains sharing admin contact street",
+                                                        "title": "Admin Contact Street Count"

```

Comment:
@RamboV please make sure that the sentences are complete and are grammatically correct.
Make sure that English language strings are in complete form. You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).